### DISCUSSION TITLE
Kann BSC auch via RS485 senden?

### DISCUSSION THREAD

**Discussion by PaulEPop** [shining-man/bsc_fw#122] [General]
Hallo wir (mein PV-Mitstreiter) kämpfen seit gut einem halben Jahr die Kommunikation zwischen Seplos (v. 10D) mit RS485 zum MPP-Solar MPI12k Wechselrichter zum laufen zu bekommen. Mit 0 Erfolg :(
Nun bin ich heute über dieses Projekt gestoßen und habe immer nur CAN-Bus gelesen, daher meine Frage nach RS485. :)
Ich selber habe das baugleiche SEPLOS via RS485 an einem Deye 12k laufen, ohne Probleme.
Habe ich hier evtl. den golden Pfad gefunden?

**Comment by shining-man** [comment]
Ich sende zur Zeit nur per CAN an den Wechselrichter. 
Zum MPI10k gibt es ein offenes Issue: https://github.com/shining-man/bsc_fw/issues/42. Hier war der Wunsch das Lesen der Werte vom Wechselrichter.

Wo liegt das Problem das Seplos am MPI12k anzubinden? 
Wenn es ein Communication Protocol gibt, dann sollte es doch eigentlich machbar sein.

**Reply by PaulEPop** [reply]
Irgendwie habe ich aber das Gefühl das beide Geräte aneinander vorbei reden.
Wir haben alle Ports des BMS und alle Protokolle des Wechselrichters durchgetestet, ohne Erfolg.
Außer beim Anschluss des Wechselrichters mit Pylon Protokoll an der BMS internen RS485 (Schnittstelle zum PC bzw. Verbindung zwischen den BMS). Hier war zumindest eine kommunikation ersichtlich, der Wechselrichter hat mit entspr. Pictogramm reagiert. Jedoch scheinen die Werte unbrauchbar, da er im Wechsel zwischen Batteriespannung zu klein und normalbetrieb hin und her schaltete.

Schade des kein RS485 geht aber trotzdem besten Dank für die Rückmeldung.
